Binance.Net/Objects/Models/Futures/BinanceFuturesBatchEditOrder.cs

```diff
@@ -30,6 +30,10 @@ public record BinanceFuturesBatchEditOrder
         /// <summary>
         /// Price
         /// </summary>
-        public decimal Price { get; set; }
+        public decimal? Price { get; set; }
+        /// <summary>
+        /// PriceMatch
+        /// </summary>
+        public PriceMatch? PriceMatch { get; set; }
```

Comment:
This is not the actual model which gets send, you'll need to copy it in the actual EditMultipleOrdersAsync methods

Reply:
It is added in the Coin futures I see, but not in the USD futures EditMultipleOrdersAsync method right?

Reply:
I think you mean the other way around :-) It's added on Usd but not Coin, I'll fix that!

Reply:
I don't see any `Edit*` methods implemented on the `BinanceRestClientCoinFuturesApiTrading`. To be clear the changes I did are on `UsdFuturesApi`, both REST and Socket. Hope it makes sense.